llama-index-core/llama_index/core/chat_engine/condense_plus_context.py

```diff
@@ -38,6 +47,23 @@
   Answer "don't know" if not present in the document.
   """
 
+DEFAULT_CONTEXT_REFINE_PROMPT_TEMPLATE = """
+  The following is a friendly conversation between a user and an AI assistant.
+  The assistant is talkative and provides lots of specific details from its context.
+  If the assistant does not know the answer to a question, it truthfully says it
+  does not know.
+
+  Here are the relevant documents for the context:
+
+  {context_str}
+
+  Existing Answer:
+  {existing_answer}
+
+  Instruction: Refine the existing answer using the provided context to assist the user.
+  Answer "don't know" if not present in the document.
```

Comment:
would this override an existing answer with don't know if the specifc new piece of context is not relevant?

Reply:
Good point, whoops

Reply:
Fixed!